c++ training/W2S2/Spotify/Spotify.cpp

```diff
@@ -0,0 +1,177 @@
+#include "Spotify.h" 
+#include <string>
+
+//Exercise 4.2 - Create a Playlist Menu
+//With the main menu done, it�s time to create the create - a - playlist menu.You should have the following private methods:
+//void openCreatePlaylistMenu();
+//
+//The method should allow the user to enter a name to the playlistand add songs to it.
+//
+//
+//Exercise 4.3 - Browser Playlists Menu
+//Let�s create the final menu for our app, the one where you can browse playlists.Create the private method:
+//void openBrowsePlaylistsMenu();
+//
+//The method should allow the user to browse the currently created playlistand choose one to be displayed.Choose how you want to make the user choose their next action.The example shows using enter as the key to go back
+
+
+void Spotify::openMainMenu()
+{
+	system("cls");
```

Comment:
Spaces are your friends. Good use of spaces can make the code more legible by separating the code into similar actions/ideas. Having this huge method with no spaces between statements makes it tedious to read.